_maps/_basemap.dm

```diff
@@ -13,8 +13,6 @@
 		#include "map_files\Mining\Lavaland.dmm"
 		#include "map_files\NorthStar\north_star.dmm"
 		#include "map_files\tramstation\tramstation.dmm"
-		#include "map_files\Birdshot\birdshot.dmm"
```

Comment:
?

Reply:
for some reason my fork has birdshot in there twice, this wouldn't be there.